test/access/manager/AuthorityUtils.test.js

```diff
@@ -64,24 +64,6 @@ describe('AuthorityUtils', function () {
       });
     });
 
-    describe('when authority replies with a delay', function () {
-      beforeEach(async function () {
-        this.authority = this.authorityDelayMock;
-      });
-
-      for (const immediate of [true, false]) {
-        for (const delay of [0n, 42n]) {
-          it(`returns (immediate=${immediate}, delay=${delay})`, async function () {
-            await this.authority._setImmediate(immediate);
-            await this.authority._setDelay(delay);
-            const result = await this.mock.$canCallWithDelay(this.authority, this.user, this.other, '0x12345678');
-            expect(result.immediate).to.equal(immediate);
-            expect(result.delay).to.equal(delay);
-          });
-        }
-      }
-    });
-
```

Comment:
This test is not so much fuzzing as it is testing 4 clearly different cases. Not saying fuzzing would not cover them, but we don't really need 5000 fuzzing runs to cover the same code.

Also, I'm not sure how that change would affect coverage.